packages/components/file-uploader-input/src/file-uploader-input.js

```diff
@@ -0,0 +1,233 @@
+			super();
+			this.multiple = false;
+			this.disableDragAndDrop = false;
+			this.size = 'full';
+			this.dragCounter = 0;
+			this.handleUpload = this.handleUpload.bind(this);
+			this.dragEnterHandler = this.dragEnterHandler.bind(this);
+			this.dragOverHandler = this.dragOverHandler.bind(this);
+			this.dragLeaveHandler = this.dragLeaveHandler.bind(this);
+			this.dropHandler = this.dropHandler.bind(this);
+		}
+
+		get dropBox() {
+			return html`
+				<div class="draggable-info drop">
+					<div class="icon-wrapper">
+						<div class="icon">
+							<svg
+								width="20px"
+								height="20px"
+								viewBox="0 0 16 16"
+								version="1.1"
+								xmlns="http://www.w3.org/2000/svg"
+								xmlns:xlink="http://www.w3.org/1999/xlink"
+							>
+								<g
+									id="Symbols"
+									stroke="none"
+									stroke-width="1"
+									fill="none"
+									fill-rule="evenodd"
+								>
+									<g
+										id="card/5.-File-card/drag-and-drop"
+										transform="translate(-113.000000, -22.000000)"
+										fill="#00B0FF"
+									>
+										<g id="5.-File-card">
+											<g
+												id="Content"
+												transform="translate(113.000000, 19.000000)"
+											>
+												<g
+													id="upload-icon"
+													transform="translate(0.000000, 3.000000)"
+												>
+													<path
+														d="M8.01904762,16 C6.55872286,16 5.21587914,15.6412734 3.99047619,14.9238095 C2.76507324,14.2063456 1.79365438,13.2349268 1.07619048,12.0095238 C0.358726571,10.7841209 0,9.44127714 0,7.98095238 C0,6.5333261 0.358726571,5.2000061 1.07619048,3.98095238 C1.79365438,2.76189867 2.76507324,1.79365438 3.99047619,1.07619048 C5.21587914,0.358726571 6.55872286,0 8.01904762,0 C9.4666739,0 10.7999939,0.358726571 12.0190476,1.07619048 C13.2381013,1.79365438 14.2063456,2.76189867 14.9238095,3.98095238 C15.6412734,5.2000061 16,6.5333261 16,7.98095238 C16,9.44127714 15.6412734,10.7841209 14.9238095,12.0095238 C14.2063456,13.2349268 13.2381013,14.2063456 12.0190476,14.9238095 C10.7999939,15.6412734 9.4666739,16 8.01904762,16 Z M8.91428571,13.1428571 L8.91428571,6.7047619 L11.7714286,9.61904762 L11.8857143,9.61904762 L13.1047619,8.36190476 C13.1301589,8.36190476 13.1428571,8.35238105 13.1428571,8.33333333 C13.1428571,8.31428562 13.1301589,8.3047619 13.1047619,8.3047619 L9.23809524,4.45714286 L8.91428571,4.11428571 L8.91428571,4.07619048 C8.91428571,4.063492 8.90793657,4.05714286 8.8952381,4.05714286 L8.83809524,4.05714286 L8.01904762,3.2 L7.92380952,3.2 L7.08571429,4.05714286 L6.7047619,4.45714286 L2.85714286,8.3047619 C2.84444438,8.3047619 2.83809524,8.31428562 2.83809524,8.33333333 C2.83809524,8.35238105 2.84444438,8.36190476 2.85714286,8.36190476 L4.07619048,9.61904762 L4.17142857,9.61904762 L7.08571429,6.66666667 L7.08571429,13.1428571 C7.08571429,13.1682541 7.09206343,13.1809524 7.1047619,13.1809524 L8.8952381,13.1809524 C8.90793657,13.1809524 8.91428571,13.1682541 8.91428571,13.1428571 Z"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 1617 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ALineLength)